libs/cloud/odc/aws/queue.py

```diff
@@ -27,8 +27,10 @@ def post_messages(to_queue, messages):
             message.delete()
         return []
 
+    if limit < 1:
```

Comment:
at this point `limit=None` is a possibility, so you need to check for that as you can't compare `None < 1`.
```suggestion
    if limit is not None and limit < 1:
```

Reply:
You are totally right. Also, limit is just used on **get_messages** which already is checking for that, so this check is useless.
Removed.
Thank you